access-common/src/main/java/edu/unc/lib/dl/ui/service/SolrQueryLayerService.java

```diff
@@ -241,6 +239,10 @@ public boolean hasRole(AccessGroupSet accessGroups, UserRole userRole) {
     }
 
     public SearchResultResponse performSearch(SearchRequest searchRequest) {
+        return performSearch(searchRequest, false);
+    }
+
+    public SearchResultResponse performSearch(SearchRequest searchRequest, boolean allResults) {
```

Comment:
Please add "allResults" as a field of SearchRequest since it is part of the request

Reply:
You'll also need to remove the method parameter here since allResults is in the search request now